### PR TITLE
Plausible tracking

### PR DESCRIPTION
**Pull request by bnelson-czi**
My attempt to add Plausible tracking to the Bravo cohort site, borrowing some code from the [docs repo](https://github.com/napari/napari/tree/main/docs). 

cc @codemonkey800

### CODE REVIEW

File: conf.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+project = 'napari'
+copyright = '2022, The napari team'
+author = 'The napari team'
```
Comment (neuromusic) [inline conf.py]:
```suggestion
```

Comment (bnelson-czi) [reply conf.py]: alternatively, could I replace these with 'CZI imaging' or something like that?

**Comment by codemonkey800** [comment]
hmm I tried this locally and it's not working as I expected, I did a ripgrep for `plausible` in the output and it's not there 🤔 

I think it might be because `_conf.py` is Sphinx specific, but Jupyterbook uses a different config. maybe @melissawm knows?

**Comment by bnelson-czi** [comment]
@codemonkey800 Not sure if this is related, but when I go through the signup flow in Plausible to enable tracking (https://plausible.io/sites -> Add a website -> etc.), it ends with this warning.

> Site locked
> This site is locked because you don't have an active subscription. We are still counting stats in the background but your access to the dashboard is restricted. Subscribe with the link below to access your stats again.

Do we need a paid account?

**Comment by codemonkey800** [comment]
hmm that I'm not sure, @sofroniewn is the owner of the account. any ideas nick?

**Comment by melissawm** [comment]
Ah I missed this. @bnelson-czi can you try updating https://github.com/chanzuckerberg/napari-segmentation-workshop/blob/main/content/_config.yml instead of using a new conf.py file? (See, for example, https://jupyterbook.org/en/stable/sphinx/index.html)

**Comment by bnelson-czi** [comment]
@melissawm Honestly not sure what to update _config.yml with, and the example suggests creating a conf.py file instead. For the record, I'm not familiar with how Jupyter Books/Sphinx works and frankly haven't done anything like this. I can take some time to learn or hand this off to someone.

**Comment by melissawm** [comment]
Ok, here's the situation:

Instead of creating a new `conf.py` file manually, you can update the `_config.yml` file adding this:

```
sphinx:
	config:
		templates_path: ['_templates']
```

This, along with your new template, should be enough to activate the plausible tracking here. However, **this is currently not working**: see https://github.com/executablebooks/jupyter-book/issues/1660

There are two options:
1. Patch the current setup with this PR, which solves the issue: https://github.com/executablebooks/sphinx-book-theme/pull/566

2. Wait for it to be merged, update the theme and activate the plausible tracking. We don't know the timeline on this, though. 

I can help with both options, just let me know!

**Comment by codemonkey800** [comment]
@bnelson-czi @neuromusic pushed a commit that fixes the config for plausible based on @melissawm's suggestions.

I also verified that plausible is added to the HTML:

image

**Comment by bnelson-czi** [comment]
sweet, thanks @codemonkey800 ! should we merge and check it out in Plausible?

**Comment by codemonkey800** [comment]
yes, it should work after this. I think there's some additional setup that needs to happen on plausible to track the data though

**Comment by codemonkey800** [comment]
I think either @neuromusic or @sofroniewn, we'll need a new website set up on plausible.org for the workshop page:

https://chanzuckerberg.github.io/napari-segmentation-workshop

**Comment by codemonkey800** [comment]
actually I just remembered I'm also signed into the plausible account so I can add it lol but I think we're going to need a new domain if we want to add it:

image

I'm pretty sure this is because Plausible checks by domain name and not by subpath, so there's probably another CZI service using plausible that's registered under the domain `chanzuckerberg.github.io`

**Comment by bnelson-czi** [comment]
I had it registered under my account. Just removed it. Try again @codemonkey800 ?

**Comment by melissawm** [comment]
Awesome - it looks like the fix went into [sphinx-book-theme v0.3.3](https://github.com/executablebooks/sphinx-book-theme/pull/566). Let me know if there's anything else I can do to help!

**Comment by neuromusic** [comment]
Thanks for releasing that @bnelson-czi !!! Got it registered!

image

We should start seeing data once this PR is merged (assuming we're using the latest theme)

**Pull request merged**